### PR TITLE
Update tuya.markdown with instructions for new user code functionality

### PR DESCRIPTION
**Pull request by codyc1515**
## Proposed change
The steps described were incomplete for me. I was not able to follow them as written. I am using the Tuya Smart app, which may be why there were different (i.e. if they were written for Smart Life).

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - [x] I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by codyc1515** [comment]
Not sure why the bot is saying that.

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -55,12 +55,15 @@ All Home Assistant platforms are supported by the Tuya integration, except the l
 You need to have the Tuya Smart or Smart Life app installed, with an account created and
 at least one device added to that account.
 
-You will need your **User Code**. You can find this code in the Smart Life app or
-Tuya Smart app in **Settings** > **Account and Security** screen, and enter the
-code shown on the **User Code** field.
+### Obtaining User Code for sign-in
+To sign-in, you will need to get your **User Code** from the Smart Life /
+Tuya Smart app. You can find it by opening the app and:
```
Comment (frenck) [inline source/_integrations/tuya.markdown]:
```suggestion
### Obtaining User Code for sign-in

To sign-in, you will need to get your **User Code** from the Smart Life /
Tuya Smart app. You can find it by opening the app and:
```

File: source/_integrations/tuya.markdown [hunk 2]
```diff
@@ -55,12 +55,15 @@ All Home Assistant platforms are supported by the Tuya integration, except the l
...
 
-Please note, that the user code is case sensitive, please be sure to enter it
-exactly as shown in the app.
+1. On the tab bar, tap **Me**
+2. In the top-right corner, tap the **⚙️ (gear)** icon
+3. Tap **Account and Security**
+4. At the bottom, **User Code** will be shown
+5. Enter this in Home Assistant exactly as shown (it is case-sensitive)
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: Removed step 5, as the user isn't there yet at this point.

```suggestion
1. On the tab bar, select **Me**.
2. Select the **⚙️ (gear)** icon in the top-right corner.
3. Tap **Account and Security**.
4. At the bottom, **User Code** will be shown; you need to when setting up this integration.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by codyc1515** [comment]
Thank you for the feedback. I have incoporated that now.

**Review by frenck — APPROVED** [review]
Thanks, @codyc1515 👍 

../Frenck

**Pull request merged**